src/components/EventForm/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import CoreTypography from "src/components/core/typography";
+import colors from "src/components/core/colors";
+import Container from "@material-ui/core/Container";
+import Link from "@material-ui/core/Link";
+import constants from "utils/constants";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+import DescriptionIcon from "@material-ui/icons/Description";
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        container: {
+            display: "flex",
+            textAlign: "center",
+            width: "750px",
+            borderStyle: "solid",
+            borderWidth: "2px",
+            borderColor: colors.green,
```

Comment:
Can you use the primary color from `theme` here? Should be `theme.palette.primary.main`.